docs/design/editorial_design.md

```diff
@@ -62,7 +40,10 @@ Let's go through some of the most common commands and what the python API might
 ## Overwrite
 
 ![Overwrite](../_static/edit/01_overwrite.png)
-- Anything overlapping is destroyed on contact. This may split an item.
+- Augment the source range of overlapping items
```

Comment:
To be honest, I still feel the wording is not as precise as it could be.  Elsewhere in OTIO we are adopting Allen's nomenclature, here, because the intent of the words is unambiguous: https://en.wikipedia.org/wiki/Allen's_interval_algebra